terragrunt/aws/api/ecr.tf

```diff
@@ -6,8 +6,38 @@ resource "aws_ecr_repository" "api" {
     scan_on_push = true
   }
 
+  lifecycle {
+    ignore_changes = [
+      # Ignore changes to tags since tags are being updated by Github workflow
+      # to track virus definition hashes.
+      tags,
+    ]
+    prevent_destroy = true
+  }
+
   tags = {
     CostCentre = var.billing_code
     Terraform  = true
   }
 }
+
+resource "aws_ecr_lifecycle_policy" "scan_files_exire_untagged" {
+  repository = aws_ecr_repository.api.name
+  policy = jsonencode({
+    "rules" : [
```

Comment:
Since we're now going to be creating a lot more docker images, do you want to also add a lifecycle rule that limits the total number of ECR images we keep?  Similar to what I tried to do in #147 before we realized it wasn't a great idea for my use case.

Reply:
Good idea. Added!